### PR TITLE
README Revised

### PR DESCRIPTION
**Pull request by dongbohu**
## Issue Number

NA

## Purpose/Implementation Notes

Changed a few wordings and corrected typos in `README.md`.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Functional tests

List out the functional tests you've completed to verify your changes work locally.

## Checklist

_Put an `x` in the boxes that apply._

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots

Please attach any screenshots that illustrate these changes.

### CODE REVIEW

**Comment by Miserlou** [comment]
Why did you add numbers to the headers? This will be difficult to maintain unless you're using a tool as we'll have to reorder the whole manually every time we add a section. It's also pretty non standard to do this and I don't think adds a whole lot.

If we do want to use a tool, I suggest we use [DocToc](https://github.com/thlorenz/doctoc) to generate Tables of Contents.

**Comment by dongbohu** [comment]
@Miserlou I deleted the numbers and added TOC with doctoc. Please take a look again. Thanks.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,37 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
```
Comment (Miserlou) [inline README.md]: You can remove the `*generated with [DocToc](https://github.com/thlorenz/doctoc)*` ad and it knows not to put it back next time doctoc is run.

**Comment by dongbohu** [comment]
@Miserlou Thanks. I have removed doctoc link and committed the change.

**Comment by dongbohu** [comment]
@Miserlou Do you have any other comments on this PR? If not, I will close it. Thanks.

**Comment by Miserlou** [comment]
Yep you can merge now

**Pull request merged**